runfo/RuntimeInfo.cs

```diff
@@ -1,21 +1,21 @@
+using DevOps.Util;
+using DevOps.Util.DotNet;
+using Mono.Options;
 using System;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
-using DevOps.Util;
-using DevOps.Util.DotNet;
-using Mono.Options;
-using static RuntimeInfoUtil;
 using static DevOps.Util.DotNet.OptionSetUtil;
-using System.Text;
+using static RuntimeInfoUtil;
 
 internal sealed partial class RuntimeInfo
 {
-    internal DevOpsServer Server { get; }
+    internal DevOpsServer DevopsServerInstance{ get; }
+
+    internal HelixServer HelixServerInstance { get; }
```

Comment:
Can we remove the `Instance` suffix from both members?

Reply:
The reason why I added instance was to differentiate in between the Property name and the Type name, but sure, I can remove the suffix from the name.

Reply:
C# can differentiate here. It's called the Color Color problem. Designed to allow situations like this work :smile: